dasp_window fails to compile with error[E0282]: type annotations needed

dasp_window fails to compile with rustc 1.50.0 / macOS 11.0.1:
```
error[E0282]: type annotations needed
  --> /Users/jsd/.cargo/registry/src/github.com-1ecc6299db9ec823/dasp_window-0.11.0/src/hanning/mod.rs:25:41
   |
25 |         let v = phase.to_float_sample().to_sample() * PI_2;
   |                                         ^^^^^^^^^ cannot infer type for type parameter `S` declared on the associated function `to_sample`
   |
help: consider specifying the type argument in the method call
   |
25 |         let v = phase.to_float_sample().to_sample::<S>() * PI_2;
   |                                                  ^^^^^

error: aborting due to previous error

For more information about this error, try `rustc --explain E0282`.
error: could not compile `dasp_window`
```

--- a/.github/workflows/dasp.yml
+++ b/.github/workflows/dasp.yml
@@ -196,7 +196,7 @@ jobs:
         args: --all --all-features --verbose
 
   cargo-publish:
-    if: github.event_name == 'push' && github.ref == 'refs/heads/master'
+    if: github.event_name == 'push' && github.ref == 'refs/heads/stable-0.11'
     env:
       CRATESIO_TOKEN: ${{ secrets.CRATESIO_TOKEN }}
     runs-on: ubuntu-latest

--- a/dasp_window/Cargo.toml
+++ b/dasp_window/Cargo.toml
@@ -1,7 +1,7 @@
 [package]
 name = "dasp_window"
 description = "Windowing function abstractions (e.g. hanning, rectangle) for audio PCM DSP."
-version = "0.11.0"
+version = "0.11.1"
 authors = ["mitchmindtree <mitchell.nordine@gmail.com>"]
 readme = "../README.md"
 keywords = ["dsp", "window", "hanning", "pcm", "audio"]

